Libraries/NetworkingLibraries.php

```diff
@@ -881,6 +881,7 @@ function ResolveSingleTarget($mainPlayer, $defPlayer, $target, $attackerPrefix,
   global $combatChain, $combatChainState, $mainPlayer, $defPlayer, $CCS_CombatDamageReplaced;
   global $CCS_DamageDealt;
 
+  $attackerSurvived = 1;
```

Comment:
you can remove this and check `!$attackerDestroyed` below (~line 939)

Reply:
oh also can initialize `$attackerDestroyed = 0` here. i had a similar fix for the Ezra bug in #448

Reply:
@mariotee Did you fix this in #448, right? It's related to Ezra's bug. So, I'll remove my changes.